transport_versioned_docs/version-1.0.0/workflow-client-server-secure.md

```diff
@@ -0,0 +1,147 @@
+---
+id: jobs
+title: Create jobified client and server
+---
+The unity transport protocol can be configure to encrypt the connection between the server and the client while ensuring the server's/client's authenticity.
+
+## Server authentication
+
+### High level authentication process
+In this configuration, the server will provide a certificate to the client (`rsa`)
+The client will then validate the provided certificate against the one its root certificate (`pem`) and will subsequently validate the server identity.
+Once its identity confirmed, the server will then use the private key (`rsaKey`) to encrypt the messages targeting the client.
+
+### Requirements
+To use the client server secure workflow, you need a valid certificate and the root certificate it has been generated from. You also need the private key that has been used to create the certificate.
+* Samples root certificate, private key and certificate are provided with the sample project. Please do not use them for production. 
+* It is of the upmost importance for you to have your own certificates. The procedure to generate them using OpenSSL is detailed hereafter. 
+
+### Generating the required keys and certificates with OpenSSL
+
+It is assumed to you have openSSL installed on your machine.
+
+#### Generate the Certification Authority root certificate. 
+First thing first is to generate a private key for that will help to generate the certification authority root certificate. 
+```shell
+openssl genrsa -des3 -out clientPrivateKeyForRootCA.key 2048
+```
+You will have to provide a passphrase. Remember it as you will need it later on.
+
+Now that you have a private key, you can now generate the root certificate.
+
+```shell
+openssl req -x509 -new -nodes -key clientPrivateKeyForRootCA.key -sha256 days 1095 -out myGameClientCA.pem
+```
+You will be prompted to answer several questions. Most of the answers are not that important within the present context. 
+It is however useful to use a `common name` that makes sense for you as it is useful to identify the certificate amongst others.
+
+
+#### Generate the CA-signed certificate to use with the server
+Create now a private key for the server. 
+```shell
+openssl genrsa -out myGameServerPrivate.key 2048
+```
+From this private key, you can generate a certificate signing request (csr). 
+```shell
+openssl req -new -key myGameServerPrivate.key -out myGameServerCertificateSigningRequest.csr
+```
+You'll be prompted with the same questions as for generating the root certificate.
+The answers are no more important. Make sure to use a proper `common name` though as it might be useful in case you need to debug the certificate.
```

Comment:
It is customary to use the domain name of your server as the common name. In fact, for web browsers it's more than customary, it's a basically a requirement.

Reply:
Added a note in this sense.